packages/source-graphql/package.json

```diff
@@ -17,7 +17,8 @@
     "node-fetch": "^2.6.0"
   },
   "peerDependencies": {
-    "gridsome": "0.x"
+    "gridsome": "0.x",
+    "graphql": "^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0"
```

Comment:
Can this be a direct dependency instead? `apollo-link-http` and `graphql-tools` have it as a peer dependency, and this package is requiring GraphQL types from `gridsome/graphql`. So it would be nice if the users don't need to install `graphql` in their projects.

Reply:
That would be possible, but then there is no guarantee that there will be one graphql version/instance. Though there isn't with this either as the user can install something different than what `gridsome` uses.

Reply:
What would you like to do here?

Reply:
I think we can add it as a direct dependency here. And maybe be more specific about the `gridsome` version listed as peer dependency to ensure this package uses the same `graphql` version as `gridsome`. Do you think that will work?

Reply:
It has a posibility to work, but it's not guaranteed to be the same version

Reply:
I've made it a direct dependency